docs/collections/documents/create-document-store.md

```diff
@@ -39,7 +44,7 @@ The below example shows the steps for connecting a fabric and then creating a co
 ```
 
 </TabItem>
-<TabItem value="js" label="Javascript">
+<TabItem value="js" label="Javascript SDK">
 
```

Comment:
The below example shows the steps for connecting a fabric and then creating a collection called `employees`.